tests/jobs/test_local_task_job.py

```diff
@@ -547,18 +550,83 @@ def task_function(ti):
         process = multiprocessing.Process(target=job1.run)
         process.start()
 
-        for _ in range(0, 10):
+        for _ in range(0, 20):
             ti.refresh_from_db()
-            if ti.state == State.RUNNING:
+            if ti.state == State.RUNNING and ti.pid is not None:
                 break
             time.sleep(0.2)
         assert ti.state == State.RUNNING
+        assert ti.pid is not None
         os.kill(ti.pid, signal.SIGTERM)
         process.join(timeout=10)
         assert failure_callback_called.value == 1
         assert task_terminated_externally.value == 1
         assert not process.is_alive()
 
+    def test_process_sigkill_call_on_failure_callback(self):
+        """
+        Test that ensures that when a task is killed with sigterm
```

Comment:
```suggestion
        Test that ensures that when a task is killed with sigkill
```

Should be sigkill, right?